test/unit/seed_helper_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require Rails.root + 'db/seeds.d/020-roles_list.rb'
```

Comment:
This is needed so `RolesList` is always available, if the new tests run before any call to `SeedHelper` this will not yet be loaded (try with `SEED=4051`)